src/tiled/changelayer.cpp

```diff
@@ -121,6 +121,29 @@ SetLayerOpacity::SetLayerOpacity(Document *document,
                                         "Change Layer Opacity"));
 }
 
+bool SetLayerTintColor::mergeWith(const QUndoCommand *other)
+{
+    const SetLayerTintColor *o = static_cast<const SetLayerTintColor*>(other);
+    if (!(mDocument == o->mDocument &&
+          mLayer == o->mLayer))
+        return false;
+
+    mNewTintColor = o->mNewTintColor;
+    return true;
+}
+
+SetLayerTintColor::SetLayerTintColor(Document *document,
+                                 Layer *layer,
+                                 QColor tintColor)
```

Comment:
Please place the constructor before the methods.

Reply:
done